corehq/apps/userreports/pillow_utils.py

```diff
@@ -18,7 +32,15 @@ def rebuild_sql_tables(adapters):
         else:
             all_adapters.append(adapter)
     for adapter in all_adapters:
-        tables_by_engine[adapter.engine_id][adapter.get_table().name] = adapter
+        if not _is_datasource_deactivated(adapter):
```

Comment:
nit: `not _is_datasource_deactivated` is harder to read than `_is_datasource_active` because of the double negative. The previous name seemed better to me.